content-src/components/DiscoveryStreamComponents/Navigation/_Navigation.scss

```diff
@@ -0,0 +1,45 @@
+.ds-navigation {
+
+  &.ds-navigation-centered {
+    text-align: center;
+  }
+
+  &.ds-navigation-right-aligned {
+    text-align: right;
```

Comment:
Hmmmmmm............ I wonder if the API should be specified as start/end instead of right/left…